source/slang/slang-ir.cpp

```diff
@@ -4495,6 +4495,18 @@ namespace Slang
         return ssboType;
     }
 
+    IRGLSLShaderStorageBufferType* IRBuilder::createGLSLShaderStorableBufferType(UInt operandCount, IRInst* const* operands)
+    {
+        IRGLSLShaderStorageBufferType* ssboType = createInst<IRGLSLShaderStorageBufferType>(
+            this,
+            kIROp_GLSLShaderStorageBufferType,
+            getTypeKind(),
+            operandCount,
+            operands);
```

Comment:
Shouldn't this be like this?
```
        IRGLSLShaderStorageBufferType* ssboType = createInst<IRGLSLShaderStorageBufferType>(
            this,
            kIROp_GLSLShaderStorageBufferType,
            getTypeKind(),
            operands[0],
            operands[1]);
```
You should try to avoid using a variable-length array as a function parameter if possible.

Reply:
Please answer to my question